artifacts/definitions/Linux/Events/ProcessStatuses.yaml

```diff
@@ -0,0 +1,62 @@
+name: Linux.Events.ProcessStatuses
+description: |
+  This artifact collects New process status logs from the Linux kernel.
+
+  This artifact relies on the presence of `auditctl` usually included
+  in the auditd package. On Ubuntu you can install it using:
+
+  ```
+  apt-get install auditd
+  ```
+
+precondition: SELECT OS From info() where OS = 'linux'
+
+type: CLIENT_EVENT
+
+required_permissions:
+  - EXECVE
+
+parameters:
+  - name: pathToAuditctl
+    default: /sbin/auditctl
+    description: We depend on auditctl to install the correct process execution rules.
+
+sources:
+  - query: |
+     // Install the auditd rule if possible.
+     // Remove default filter if any
+     LET _ <= SELECT * FROM execve(argv=[pathToAuditctl, "-d", "task,never"])
+
+     LET _ <= SELECT * FROM execve(argv=[pathToAuditctl, "-a",
+          "exit,always", "-F", "arch=b64", "-F", "auid!=-1", "-F",
+          "euid!=0", "-S", "execve", "-k", "proexec"])
```

Comment:
Suggest prefix the tag with `vrr_` to distinguish the ones added by velociraptor. i.e. `vrr_proexec`.

Reply:
Agreed :)